### ISSUE TITLE
No dependencies com.alphacephei:vosk:0.3.37

### ISSUE THREAD

**Issue by KingBoyAndGirl** [alphacep/vosk-api#916]
15:51:26: Executing task 'dependencies'...

> Task :dependencies

------------------------------------------------------------
Root project 'demo'
------------------------------------------------------------

annotationProcessor - Annotation processors and their dependencies for source set 'main'.
No dependencies

apiElements - API elements for main. (n)
No dependencies

archives - Configuration for archive artifacts. (n)
No dependencies

compileClasspath - Compile classpath for source set 'main'.
+--- net.java.dev.jna:jna:5.7.0
\--- com.alphacephei:vosk:0.3.37+ FAILED

compileOnly - Compile only dependencies for source set 'main'. (n)
No dependencies

default - Configuration for default artifacts. (n)
No dependencies

implementation - Implementation only dependencies for source set 'main'. (n)
+--- net.java.dev.jna:jna:5.7.0 (n)
\--- com.alphacephei:vosk:0.3.37+ (n)

runtimeClasspath - Runtime classpath of source set 'main'.
+--- net.java.dev.jna:jna:5.7.0
\--- com.alphacephei:vosk:0.3.37+ FAILED

runtimeElements - Elements of runtime for main. (n)
No dependencies

runtimeOnly - Runtime only dependencies for source set 'main'. (n)
No dependencies

testAnnotationProcessor - Annotation processors and their dependencies for source set 'test'.
No dependencies

testCompileClasspath - Compile classpath for source set 'test'.
+--- net.java.dev.jna:jna:5.7.0
\--- com.alphacephei:vosk:0.3.37+ FAILED

testCompileOnly - Compile only dependencies for source set 'test'. (n)
No dependencies

testImplementation - Implementation only dependencies for source set 'test'. (n)
No dependencies

testRuntimeClasspath - Runtime classpath of source set 'test'.
+--- net.java.dev.jna:jna:5.7.0
\--- com.alphacephei:vosk:0.3.37+ FAILED

testRuntimeOnly - Runtime only dependencies for source set 'test'. (n)
No dependencies

(n) - Not resolved (configuration is not meant to be resolved)

A web-based, searchable dependency report is available by adding the --scan option.

BUILD SUCCESSFUL in 18s
1 actionable task: 1 executed
15:51:46: Task execution finished 'dependencies'.

**Comment by nshmyrev** [comment]
What are your dependency sources?

**Comment by KingBoyAndGirl** [comment]
How to configure in Maven to load vosk: 0.3.37?

**Comment by KingBoyAndGirl** [comment]
My current configuration is:

```java
<repositories>
        <repository>
            <id>vosk</id>
            <url>https://alphacephei.com/maven/</url>
        </repository>
    </repositories>
 <dependencies>
        <dependency>
            <groupId>com.alphacephei</groupId>
            <artifactId>vosk</artifactId>
            <version>0.3.33</version>
        </dependency>
        <dependency>
            <groupId>net.java.dev.jna</groupId>
            <artifactId>jna</artifactId>
            <version>5.7.0</version>
        </dependency>

    </dependencies>
```


When I tried vosk: 0.3.37, I couldn't load dependencies, and 0.3.33 did.
What is the reason

**Comment by nshmyrev** [comment]
We didn't release java package 0.3.37 yet, only vosk-android 0.3.37. You can stay with 0.3.33

**Comment by KingBoyAndGirl** [comment]
Thank you very much for solving my problem

**Comment by nshmyrev** [comment]
We released 0.3.38
